### ISSUE TITLE
Russian documentation

### ISSUE THREAD

**Issue by dustalov** [bigartm/bigartm#247]
Hello,

I have written end-user documentation in Russian on NLPub: <https://nlpub.ru/BigARTM>. In order to avoid possible conflicts and collisions with the BigARTM pages on MachineLearning.ru, only beginner's guides will be maintained. It leaves me with two questions.

1. How to run a basic distributed setup? Are you still using ØMQ?
2. Could you please add a link to us to the BigARTM's documentation?

Thanks!

**Comment by sashafrey** [comment]
Great, thank you! I've added a link at http://docs.bigartm.org/en/latest/intro.html, under "Getting help" callout on the right-hand side of the page. I see that you are referencing cppclient - this is still correct, however from withing next month we expect to release a better CLI, described [here](http://nbviewer.ipython.org/github/bigartm/bigartm-book/blob/master/junk/cli/BigARTM_CommandLineInferface.ipynb).

I'm sorry to say that, but distributed setup had been removed in v0.7.0. More details behind this decision are in the [release notes](http://docs.bigartm.org/en/latest/release_notes/v0.7.0.html). We do plan to bring it back, but it will probably take some time. Current expectation is that it will be still based on ZeroMQ, plus [Parameter Server](http://parameterserver.org/). More ideas are written in #78.

How big tasks do you plan to use BigARTM for, in terms of |W|, |D| and NNZ? All scenarios we've encountered so far fitted in single machine. If this doesn't work for you please describe the scenario at our discussion group (https://groups.google.com/group/bigartm-users) and we will look into that... Thanks!

**Comment by sashafrey** [comment]
Could you add few lines to the documentation nlpub.ru? It would be great to mention that
* on Windows, BigARTM has binary distributions, regularly published at [here](https://github.com/bigartm/bigartm/releases)
* BigARTM is also available from Python, via [artm_model](http://nbviewer.ipython.org/github/bigartm/bigartm-book/blob/master/BigARTM_example_RU.ipynb) package (available in BigARTM disctibution for Windows, and included together with BigARTM's source code if you fetch it from github). BigARTM tutorials in the documentation describe how to get started with Python API.

**Comment by dustalov** [comment]
Thank you for such a quick answer! I updated the article. The new CLI example will be added when it will be released.

My tasks are quite small and I am OK with single node mode. However, I know several Russian companies that need large scale topic modeling on gigabytes of textual documents (I forgot the actual parameters). At the moment, they use various sophisticated SVD implementations that are difficult to distribute across multiple cluster nodes.

I will be watching you.

**Comment by sashafrey** [comment]
On Wikipedia corpus BigARTM usd 4.5 minutes per iteration (using 8 cores). That's for 3.8m docs, 100k tokens, 500m NNZ. |T|=100 topics. Memory usage was 3 GB. Scaling is as follows:
* |T|=1000 means 10x more time and 10x more memory
* 10x NNZ require 10x time; |D| and |W| doesn't directly influence inference time (so only NNZ matters)
* 10x dictionary (e.g. |W|) require 10x memory
* 32 cores would be 4 times faster, at no memory cost

We've tested BigARTM at 5000 topics, and it worked as long as we gave it enough memory. The largest dictionary was aroung 500k tokens. So far this is all performance numbers that I have on top of my head.

Should I close this ticket since BigARTM documentation is updated?
